src/DIRAC/MonitoringSystem/Client/DataOperationSender.py

```diff
@@ -0,0 +1,82 @@
+import DIRAC
```

Comment:
Add a docstring at the beginning of this module that explain briefly what's this about.